src/test/java/com/wikia/webdriver/testcases/specialpagestests/NewPagesTests.java

```diff
@@ -0,0 +1,22 @@
+package com.wikia.webdriver.testcases.specialpagestests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialNewPages;
+import org.testng.annotations.Test;
+
+@Execute(onWikia = "sustainingtest")
+@Test(groups = {"newpages"})
+public class NewPagesTests extends NewTestTemplate {
+  @Test
+  public void newArticleShowsUpOnSpecialNewPages() {
+    ArticleContent articleContent = new ArticleContent();
+    String articleTitle = articleContent.createUniqueArticle();
+
+    SpecialNewPages specialNewPages = new SpecialNewPages();
+
+    specialNewPages.openSpecialNewPages(wikiURL);
+    specialNewPages.verifyContainsLinkToArticle(articleTitle);
```

Comment:
If possible, please use an assertion explicite in tests, don't wrap a check into page-specific method